### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by kristi-h**
get rid of project instructions and add README for project

### CODE REVIEW

**Comment by MatthewNeie** [comment]
Carlos has a PR with changes to these files also. Not sure which one we should merge.

**Review by singhalex — APPROVED** [review]

**Comment by ccespedes** [comment]
Seems good just waiting for Kris's final tweaks and green light to merge.

**Review by ccespedes — APPROVED** [review]

**Pull request merged**
